examples/make.mk

```diff
@@ -27,6 +27,9 @@ endif
 # Board within family
 ifeq ($(BOARD_PATH),)
   BOARD_PATH := $(subst $(TOP)/,,$(wildcard $(TOP)/hw/bsp/*/boards/$(BOARD)))
+ifneq ($(wildcard $(TOP)/hw/bsp/*/boards/$(BOARD)/board.mk),)
+    include $(TOP)/hw/bsp/*/boards/$(BOARD)/board.mk
+endif
```

Comment:
the `board.mk` should not be included here. The reason is that depending on the family, it may need some variables define before the include of board.mk (e.g board linker overwrite family linker etc.. ) and some after. So it is best to remove and let the family.mk to decide.

Reply:
> 
> 
> the `board.mk` should not be included here. The reason is that depending on the family, it may need some variables define before the include of board.mk (e.g board linker overwrite family linker etc.. ) and some after. So it is best to remove and let the family.mk to decide.
This approach with included `board.mk` here was used in previous approach (with esp32sx family) to override the chip target settings. In this update it is not required any more. I removed the `board.mk` from espressif chip board folders but forgot to remove these lines from `make.mk`.